src/main/java/seedu/address/commons/core/item/Item.java

```diff
@@ -240,19 +240,19 @@ public String toString() {
         public ItemBuilder() {}
 
         public ItemBuilder setTask(Task task) {
-            requireNonNull(task);
+            //requireNonNull(task);
```

Comment:
I think it's better to delete the code rather than comment it out

Reply:
Ok! However, I choose to comment it out as it is @blimyj code, so should leave it for him to remove it.